zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/TestSupport.java

```diff
@@ -19,7 +19,7 @@
 import org.testcontainers.utility.DockerImageName;
 
 /** Collection of utilities for unit and integration tests. */
-final class TestSupport {
+public final class TestSupport {
```

Comment:
I expect this project to be probably deprecated at some points, so maybe we want to copy the little parts we need from this class in a similar one inside the scope of our project?